File: python/01-sequencias/main013.py
'''Faça um algoritmo que leia o salário de um funcionário, calcule e mostre o
seu novo salário, com 15% de aumento.

Data 28/09/2025 -> 7ª Dia
'''

salario = float(input('Digite o Salario do Funcionario: '))

aumennto = salario * 0.15

novo_salario = salario + aumennto

print(f'O salario do funcionario era {salario} com o aumento de 15% agora é: {novo_salario}')

File: c/01-sequencias/main013.c
/*Faça um algoritmo que leia o salário de um funcionário, calcule e mostre o
seu novo salário, com 15% de aumento.

Data 28/09/2025 -> 7ª Dia
*/

#include <stdio.h>

int main() {

    float salario;
    float aumento;
    float novo_salario;

    scanf("%f",&salario);

    aumento = salario * 0.15;
    novo_salario = salario + aumento;

    printf("O salario do funcionario era %f com o aumento de 15 porcento agora e: %f",salario,novo_salario);
    return 0;
}